ignite-base/App/Themes/ApplicationStyles.js

```diff
@@ -34,13 +34,15 @@ const ApplicationStyles = {
       color: Colors.snow,
       marginVertical: Metrics.smallMargin,
       textAlign: 'center',
-      fontFamily: Fonts.type.bold
+      fontFamily: Fonts.type.bold,
+      backgroundColor: Colors.transparent
     },
     subtitle: {
       color: Colors.snow,
       padding: 5,
       marginBottom: 5,
-      marginHorizontal: Metrics.smallMargin
+      marginHorizontal: Metrics.smallMargin,
+      backgroundColor: Colors.transparent
```

Comment:
Just found out if you set the `contentContainerStyle` background to transparent it fixes this.... no clue.   Let's do that instead, though, rather than affecting every text item.